apps/console/src/features/branding/providers/branding-preference-provider.tsx

```diff
@@ -99,19 +100,26 @@ const BrandingPreferenceProvider: FunctionComponent<BrandingPreferenceProviderPr
         CustomTextConfigurationModes
     >(CustomTextConfigurationModes.TEXT_FIELDS);
 
+    const [ isCustomTextPreferenceConfigured, setIsCustomTextPreferenceConfigured ] = useState<boolean>(true);
```

Comment:
Updating this state variable will result in re-rendering the entire branding section, therefore it'd be better if we can avoid the state variable and directly use the `organizationType === OrganizationType.SUBORGANIZATION
            && customText?.name !== currentOrganization?.id` condition in the places where `isCustomTextPreferenceConfigured` is used.

In that way, we can remove both `isCustomTextPreferenceConfigured` state variable and the `useEffect`.